flake.nix

```diff
@@ -0,0 +1,49 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    crane.url = "github:ipetkov/crane";
+    crane.inputs.nixpkgs.follows = "nixpkgs";
+
+    rust-overlay = {
```

Comment:
Could we consider using [fenix](https://github.com/nix-community/fenix/tree/main) instead? rust-overlay ends up being larger and I find it much more cumbersome to use

Reply:
I've switched to fenix--good idea.